Code/GraphMol/Substruct/vf2.hpp

```diff
@@ -479,43 +458,95 @@ namespace boost{
       VF2SubState *Clone(){
```

Comment:
I think this function can be removed since I has never been used in this version.